tensorflow/lite/micro/examples/person_detection_experimental/main_functions.cc

```diff
@@ -42,7 +42,9 @@ TfLiteTensor* input = nullptr;
 
 // An area of memory to use for input, output, and intermediate arrays.
 constexpr int kTensorArenaSize = 125 * 1024;
+#pragma Bss(".tensor_arena")
```

Comment:
Is this required, or just a nice to have? If it is required, it would be good to have a comment above noting why it's needed.

Reply:
It ease operating with this particular array in linker command files. But in general it's nice to have. I removed it to not accidently affect other platforms.